packages/react/src/components/tabs/tabs.tsx

```diff
@@ -15,27 +15,24 @@ import { getNextElement, getPreviousElement } from '../../utils/array';
 import { v4 as uuid } from '../../utils/uuid';
 import { Icon, IconName } from '../icon/icon';
 import { Tooltip } from '../tooltip/tooltip';
+import { Button } from '../buttons/button';
 import { TabButton } from './tab-button';
 import { TabPanel } from './tab-panel';
-import { Button } from '../buttons/button';
+import { TabSize } from './types';
+import { tabsClasses } from './tabs-classes';
+import { focus } from '../../utils/css-state';
 
-const TabsWrapper = styled.div<{ $global?: boolean; }>`
-    ${({ theme, $global }) => !$global && css`
-        background: ${theme.component['tab-section-background-color']};
-        border: 1px solid ${theme.component['tab-section-border-color']};
-        border-radius: var(--border-radius-2x);
-        box-shadow: 0 1px 4px 0 ${theme.component['tab-section-box-shadow-color']};
-    `};
-`;
+const getButtonSize = (): string => 'var(--size-2x)';
```

Comment:
Est-ce que c'est parce que t'anticipe que la taille du bouton change selon un prop bientôt? Sinon tu pourrais juste avoir une constante si tu veux pas de répéter plus bas.
```suggestion
const buttonSize = 'var(--size-2x)';
```